src/xunit.performance.api/Profilers/Etw/UserProvider.cs

```diff
@@ -24,7 +24,11 @@ static UserProvider()
                 },
                 new UserProvider {
                     Guid = ClrTraceEventParser.ProviderGuid,
-                    Keywords = (ulong)(ClrTraceEventParser.Keywords.Exception | ClrTraceEventParser.Keywords.GC | ClrTraceEventParser.Keywords.Jit | ClrTraceEventParser.Keywords.JittedMethodILToNativeMap | ClrTraceEventParser.Keywords.Loader),
+                    Keywords = (ulong)(ClrTraceEventParser.Keywords.Exception
```

Comment:
Why did we add the NGen flag and remove the JittedMethodILToNativeMap flag?

Reply:
It was a suggestion from @AndyAyersMS. We are looking at ngen data, and currently we were setting "JittedMethodILToNativeMap" and not really using it.

Reply:
If you think we need it, then I can add it back. I was just trimming flags that we are not using, and makes the generated etl bigger.

Reply:
OK, sounds good.

Reply:
We don't need the IL to native maps for our current analysis.